hmri_create_MTProt.m

```diff
@@ -403,12 +409,44 @@
                 % been added, the offset has changed and must be updated before
                 % writing the data to the file!)
                 Nmap(ccon) = nifti(Pte0{ccon});
+                if mpm_params.errormaps
+                    PR2s_OLS_error{ccon}    = fullfile(calcpath,[outbasename '_' mpm_params.input(ccon).tag 'w_errorESTATICS' '.nii']);
+                    Ni = hmri_create_nifti(PR2s_OLS_error{ccon},V_pdw(1),dt,['Error map for ' mpm_params.input(ccon).tag 'w contrast']);
+                    NEmap(ccon) = Ni;
+                    
+                    % set metadata - just copied from above...
+                    input_files = mpm_params.input(ccon).fnam;
+                    Output_hdr = init_mpm_output_metadata(input_files, mpm_params);
+                    Output_hdr.history.output.imtype = Ni.descrip;
+                    Output_hdr.history.output.units = 'a.u.';
+                    set_metadata(PR2s_OLS_error{ccon},Output_hdr,mpm_params.json); 
+                    % re-load the updated NIFTI file (in case extended header has
+                    % been added, the offset has changed and must be updated before
+                    % writing the data to the file!)
+                    NEmap(ccon) = nifti(PR2s_OLS_error{ccon});
+                    
+                end
             else
                 Pte0{ccon} = '';
+                if mpm_params.errormaps
+                    PR2s_OLS_error{ccon} = '';
+                end
             end
         end
     end % init nifti objects for fullOLS case
-    
+
+    if mpm_params.errormaps
+        ccon = ccon + 1;
+        PR2s_OLS_error{ccon}    = fullfile(calcpath,[outbasename '_' 'R2s_errorESTATICS' '.nii']);
+        Ni = hmri_create_nifti(PR2s_OLS_error{ccon},V_pdw(1),dt,'Error map for R2s contrast');
+        NEmap(ccon) = Ni;
+        
+        % this is unused
```

Comment:
I think it would be useful to leave it in because it allows combining R2* maps retrospectively. But, it everyone prefers leaving it out, I am happy to remove it.

Reply:
@lukeje Remove